ModelBundle/DataFixtures/MongoDB/DemoContent/HomeNodeGenerator.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent;
+
+use OpenOrchestra\ModelBundle\Document\Block;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\TinyMCEWysiwygStrategy;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+
+/**
+ * Class HomeNodeGenerator
+ */
+class HomeNodeGenerator extends AbstractDataGenerator
```

Comment:
its generates a empty node for the home page or HomeDataGenerator generate dummy contents